_posts/2023-01-02-release.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Release 2023.01
+author: priya
+date: 2023-01-02
+category: Jekyll
+layout: post
+---
+
+### Defect fixes
+
+* Column browser was enabled when there was a pipeline error.
```

Comment:
The column browser should stay disabled when the pipeline has an error. This has been fixed now